### ISSUE TITLE
[Request]: create_frame_tops.csh should accept relative paths in SAFE.list

### ISSUE THREAD

**Issue by ericlindsey** [gmtsar/gmtsar#89]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is, with what dataset, ...

When running create_frame_tops.csh, the script fails during make_s1a_tops if the .SAFE folder paths in SAFE.list are given as relative instead of absolute paths. The error message is cryptic, stating that a certain xml file couldn't be opened. This occurs because the create_frame_tops.csh script created an invalid link to that file, by using the relative path but from a different directory.

**Describe the solution you'd like**

Ideally, create_frame_tops.csh should check whether the paths are relative or absolute and make sure that the links created are valid.

**Describe alternatives you've considered**

Alternatively, the help message at the top of the script could be modified to clarify that absolute paths are required.

**Additional context**

**Note**
Feature requests usually mean writing codes/scripts, so expect this to be solved in the long-run.

**Comment by Xiaohua-Eric-Xu** [comment]
The script has a note in the end that says `Files listed in SAFE_filelist should be the absolute path`. It was written a while ago and I forgot why it has to be absolute path, but as you said, I met the same problem when creating links with relative paths. I am not sure whether it's a good way by just checking if the path starts with `/`, and then change the relative paths to absolute inside the script. It'll require some testing.

**Comment by Xiaohua-Eric-Xu** [comment]
Great to know! Will this get you to the real path in case the file is a symbolic link?
Also, do you think we should bury this in the organizing script or leave it outside for users to do a conversion beforehand?

**Comment by chintals** [comment]
Yes, this will work with symbolic links as well, I have tested on Ubuntu 20, am sure will work on most of the linux distros.

We could bury it in the organizing script, since any input (absolute, relative, or symbolic) will yield an absolute path. Need to test it on mac and windows though.
